includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -399,6 +400,48 @@ public function maybe_hide_payments_menu() {
 		}
 	}
 
+	/**
+	 * Hide the Customizer menu item when a block theme is used.
+	 *
+	 * Keep in mind that if there is custom css, a new menu item will appear `Additional CSS`
+	 * so there is no need to handle this case.
+	 *
+	 * @since x.x.x
+	 */
+	public function maybe_remove_customizer_menu() {
+
+		global $submenu;
+
+		if ( ! wp_is_block_theme() ) {
+			return;
+		}
+
+		if ( isset( $submenu['themes.php'] ) ) {
+			foreach ( $submenu['themes.php'] as $item ) {
+				if ( isset( $item[2] ) && strpos( $item[2], 'customize.php' ) !== false ) {
+					remove_submenu_page( 'themes.php', $item[2] );
+				}
+			}
+		}
+
+	}
+
+	/**
+	 * Remove the Customizer node from the admin bar when a block theme is used.
+	 *
+	 * @since x.x.x
+	 * @param WP_Admin_Bar $wp_admin_bar
+	 */
+	public function maybe_remove_customizer_admin_bar_menu( $wp_admin_bar ) {
+
+		if ( ! wp_is_block_theme() ) {
```

Comment:
Better use the `wc_current_theme_is_fse_theme()` here. It's available from WC v6+

Reply:
I'd also ensure that the function exists in case provision-related code happens before Woo is installed or activated.

Reply:
Thanks, today I learned. 

This file is included only when WooCommerce is active, and no provision related stuff is happening at this point